### PR TITLE
Fix timers decoding, fix mypy checks, renamed to lowercase (mtaircompressor).

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (couger01) [inline .pre-commit-config.yaml:1]: This file can be removed with `git rm --cached`.

File: SConstruct
Comment (couger01) [inline SConstruct:1]: This file along with scons plumbing can be removed.

File: doc/manifest.yaml
Comment (couger01) [inline doc/manifest.yaml:1]: This file is not necessary.

**Review by couger01 — APPROVED** [review]
LGTM. Just a few files that could be cleaned up.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 1]
```diff
@@ -253,7 +258,7 @@ async def end_start(self, data: salobj.type_hints.BaseMsgType) -> None:
             sock = [
                 s for s in self.simulator.server.sockets if s.family == socket.AF_INET
             ][0]
-            self.host, self.port = socket.getnameinfo(sock.getsockname(), 0)  # type: ignore
+            self.host, self.port = socket.getnameinfo(sock.getsockname(), 0)
```
Comment (fritzm) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Are all consumers of `.port` able to accommodate str named ports?  If not, might be better to keep typing at just `int|None` and validate/convert here?

**Review by fritzm — APPROVED** [review]
LGTM -- one minor q/comment

File: SConstruct
Comment (pkubanek) [reply SConstruct:1]: This is needed for Conda build - removing it crashed conda build.

File: .pre-commit-config.yaml
Comment (pkubanek) [reply .pre-commit-config.yaml:1]: Removed.

File: doc/manifest.yaml
Comment (pkubanek) [reply doc/manifest.yaml:1]: removed.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: changed to int only

File: SConstruct
Comment (pkubanek) [reply SConstruct:1]: Can be removed, so removed after all. Need to remove it in ts_criopy as well.

**Pull request merged**